packages/arpa_integration_tools/simple_test.json5

```diff
@@ -0,0 +1,21 @@
+{
+  "srcPath": "/Users/mattb/Desktop/usdr/arpa-reporter",
+  "srcRepoName": "arpa-reporter",
+  "destPath": "/Users/mattb/Desktop/usdr/usdr-gost",
+  "copies": {
+    "src/server/use-request.js": "packages/server/src/arpa-reporter",
+    "src/server/environment.js": "packages/server/src/arpa-reporter",
```

Comment:
I noticed that this test file uses the directory `arpa-reporter`, while `arpa-to-gost.json5` uses `arpa_reporter`.

Obviously only one of those will survive the real migration, but it might be nice to make these consistent just to avoid confusion.

Reply:
Ah yeah. I might just delete this file anyhow.